website/scripts/get-bug-and-pr-report.js

```diff
@@ -86,6 +103,35 @@ module.exports = {
           let timeOpenInMS = Math.abs(todaysDate - issueOpenedOn);
           // Convert the miliseconds to days and add the value to the daysSinceBugsWereOpened array
           let timeOpenInDays = timeOpenInMS / ONE_DAY_IN_MILLISECONDS;
+          if (timeOpenInDays <= 7) {
```

Comment:
Normally wouldn't do this kind of nested if/else but it's quick and dirty and works.